src/containers/lock/containers/LockForm/index.js

```diff
@@ -167,8 +167,12 @@ class LockContainer extends React.Component {
                                                     <TermTableCell {...{ dashboard }} style={{ textAlign: "right" }}>
                                                         {this.props.lockAmount &&
                                                             `${Math.floor(
-                                                                this.props.lockAmount * product.perTermInterest * 100
+                                                                // this.props.lockAmount * product.perTermInterest * 100
+                                                                this.props.lockAmount * product.interestRatePa * 100 / 365 * product.durationInDays
```

Comment:
1. this is wrong, the interest amount calculation was correct, it's lockAmount x perTermInterest . We just need to display p.a. interest for the user
2. it shouldn't be calculated by the UI (as p.a. interest doens't depend on the lock amount entered). It should go down to the redux store under the lock product attributes (`modules/ethereum/loanTransactions.js parseLoans()` fx) I would use a name like `interestPa`

Reply:
1) 
- (I think it was off here and there, although i might not know something. and that is why i did not use the `permTermInterest` cause that seemed to be off a bit)
- but the interest p.a. is already displayed OR i do not know what u mean.

2)
 - i though HERE it is calculated only to be displayed. The id of the product & the amount are what we send

Reply:
1. `permTermInterest` is good as it was. you need it on the form because that's the simple caluation for the interest amount based on term & lock amount entered: 
2. the `interestPa` is independent from the amount and the formula is `permTermInterest / PPM_DIV / term * 365` - it needs to be rouned to 2 decimals so it's displayed as 14.12%
As you see it's independent from the lock amount therefore it can be calculated in the reducer and stored in redux store for each product

NB: make sure you use @gyurka00 's recent decimal stripping method (he introduced it in #278 )  to avoid javascript float issues